index.html

```diff
@@ -70,5 +70,198 @@ <h2 class="first-title">I'm Peter <br/> Glad to see you! </h2>
+                <li class="role"> Back End Dev</li>
+                <li> <img src="/Images/Counter.png" alt="counter"/></li>
+                <li class="year"> 2015 </li>
+            </ul>
+            <p class="description3">A daily selection of privately personalized reads; no accounts or sign-ups required.</p>
+            <ul class="grid">
+                <li class="tag html-tag">html</li>
+                <li class="tag css-tag">css</li>
+                <li class="tag js-tag">javascript</li>
+            </ul>
+            <div class="works-button ">
+                <button type="button" class="normal-button1">See Project</button>
+            </div>
+        </div>
+    </div>
+    <div class="card card-works-layout3">
+        <a href="#">
+            <img src="/Images/SnapshootPortfolio3.png" alt="SnapshootPortfolio3" class="snapshot3">
+        </a>
+        <div class="card-works-content3">
+            <h3 class="multi-post">Multi-Post Stories</h3>
+            <ul class="ul-frame1 flex">
+                <li class="client">CANOPY</li>
+                <li><img src="/Images/Counter.png" alt="counter"/></li>
+                <li class="role"> Back End Dev</li>
+                <li> <img src="/Images/Counter.png" alt="counter"/></li>
+                <li class="year"> 2015 </li>
+            </ul>
+            <p class="description3">A daily selection of privately personalized reads; no accounts or sign-ups required.</p>
+            <ul class="grid">
+                <li class="tag html-tag">html</li>
+                <li class="tag css-tag">css</li>
+                <li class="tag js-tag">javascript</li>
+            </ul>
+            <div class="works-button ">
+                <button type="button" class="normal-button1">See Project</button>
+            </div>
+        </div>
+    </div>
+    <div class="card card-works-layout4">
+        <a href="#">
+            <img src="/Images/SnapshootPortfolio4.png" alt="SnapshootPortfolio4" class="snapshot4">
+        </a>
+        <div class="card-works-content4">
+            <h3 class="multi-post">Multi-Post Stories</h3>
+            <ul class="ul-frame1 flex">
+                <li class="client">CANOPY</li>
+                <li><img src="/Images/Counter.png" alt="counter"/></li>
+                <li class="role"> Back End Dev</li>
+                <li> <img src="/Images/Counter.png" alt="counter"/></li>
+                <li class="year"> 2015 </li>
+            </ul>
+            <p class="description3">A daily selection of privately personalized reads; no accounts or sign-ups required.</p>
+            <ul class="grid">
+                <li class="tag html-tag">html</li>
+                <li class="tag css-tag">css</li>
+                <li class="tag js-tag">javascript</li>
+            </ul>
+            <div class="works-button ">
+                <button type="button" class="normal-button1">See Project</button>
+            </div>
```

Comment:
- [ **OPTIONAL** ] **IMPORTANT:** To make your buttons stick to the ones from Figma, I suggest you this code in your `style.css` file like on the **line 8** it would be better.
```
button {
  outline: none;
  border: none;
}
```